package.json

```diff
@@ -6,8 +6,8 @@
     "node": "20.x"
   },
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
+    "dev": "npx prisma generate && next dev",
```

Comment:
nit: possibly optimize this by pulling into a separate command/avoid slowing down dev mode ⚡